### ISSUE TITLE
performance queryAvailablePerfMetric on wmfs datastore throws NullPointerException

### ISSUE THREAD

**Issue by pcoder99** [yavijava/yavijava#178]
working on vCenter 6.0.0.20000
I tried passing different VMFS datastore object into the API, all of them throws NullPointerException  ? Can anybody share a workable sample code?

Solution 1:
HostSystem host = (HostSystem)navigator.searchManagedEntity("HostSystem", "***_**");
Datastore[] ds = host.getDatastores();
if(ds != null && ds.length != 0){
            for(Datastore ads: ds){
                if(ads.getName().equals(dsName)){
                    PerfMetricId[] pmis2 = pm.queryAvailablePerfMetric(__ads_*, null, null, null);
                    break;
                }

```
        }
```

}

Solution 2:
HostDatastoreSystem dss = host.getHostDatastoreSystem();
Datastore[] ds = host.getDatastores();
PerfMetricId[] pmis22 = pm.queryAvailablePerfMetric(ds[0], null, null, null);

Solution 3:
ManagedEntity me = navigator.searchManagedEntity("Datastore", "avmfsds");
PerfMetricId[] pmis22 = pm.queryAvailablePerfMetric(ds[0], null, null, null);

**Comment by pcoder99** [comment]
Yes, it is a question. Can anybody help to answer the question ?

**Comment by michaelrice** [comment]
No reproduction details were provided and there has been no activity since 2016. If you are still seeing a NullPointerException with `queryAvailablePerfMetric`, please reopen with a minimal reproducible example against the current release. Closing as stale.
